code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -241,6 +241,76 @@ datum/admins/proc/print_chemical_reactions()
 	required_reagents = list("carbon" = 15, "hydrogen" = 12)
 	result_amount = 25
 
+/datum/chemical_reaction/ethanoic_acid
+	name = "Ethanoic Acid"
+	id = "ethanoic_acid"
+	result = "ethanoic_acid"
+	required_reagents = list("carbon" = 2, "hydrogen" = 4, "oxygen" = 2)
+	result_amount = 8
+
+/datum/chemical_reaction/calcium_chloride
+	name = "Calcium Chloride"
+	id = "calcium_chloride"
+	result = "calcium_chloride"
+	required_reagents = list("calcium carbonate" = 1, "chlorine" = 2)
+	result_amount = 3
+
+/datum/chemical_reaction/gluconic_acid
+	name = "Gluconic Acid"
+	id = "gluconic_acid"
+	result = "gluconic_acid"
+	required_reagents = list("carbon" = 6, "hydrogen" = 12, "oxygen" = 7)
+	result_amount = 25
+
+/datum/chemical_reaction/calcium_gluconate
+	name = "Calcium Gluconate"
+	id = "calcium_gluconate"
+	result = "calcium_gluconate"
+	required_reagents = list("calcium" = 1, "carbon" = 12, "hydrogen" = 22, "oxygen" = 14)
+	result_amount = 3
+
+/datum/chemical_reaction/sodium_hydroxide
+	name = "Sodium Hydroxide"
+	id = "sodium_hydroxide"
+	result = "sodium_hydroxide"
+	required_reagents = list("sodium" = 2, "sulfur" = 1)
```

Comment:
This should be 2 sodium, 1 hydrogen, 1 oxygen (I had it wrong in my issue, sorry!)